docs/module.rst

```diff
@@ -0,0 +1,267 @@
+.. _module-page:
+
+*******
+Modules
+*******
+
+.. note::
+    Modules were introduced in DSL2. If you are still using DSL1, see the :ref:`dsl1-page` page to
+    learn how to migrate your Nextflow pipelines to DSL2.
+
```

Comment:
I wonder if this is even needed. Maybe keep it, but drop it down below the _Functions, processes, and workflows_ note below? Just so that it's not the first thing you see on the page.

Reply:
Fair enough